policy_tests/run_unit_tests.py

```diff
@@ -13,32 +13,43 @@
 # in this function, a set of policy test parameters is checked
 #   to make sure that the test makes sense. If it doesnt, the
 #   function returns the reason why
-def incompatibleReason(test, policy):
+def incompatibleReason(test, policy, arch):
     # skip negative tests that are not matched to the correct policy
     if "ripe" not in test and "/" in test and (not test.split("/")[0] in policy):
         return "incorrect policy to detect violation in negative test"
     if "ppac" in policy and policy not in ["osv.bare.main.heap-ppac-userType",
-                                           "osv.frtos.main.heap-ppac-userType"]:
+                                           "osv.bare64.main.heap-ppac-userType",
```

Comment:
Again, the architecture shouldn't be in the policy module name.

Reply:
Resolved by runtime/arch split.